contrib/endpoints/include/api_manager/api_manager.h

```diff
@@ -87,6 +87,12 @@ class ApiManager {
   virtual utils::Status GetStatistics(
       ApiManagerStatistics *statistics) const = 0;
 
+  // Returns the config loading status.
+  //  - Code::UNAVAILABLE Not initialized yet. The default value.
+  //  - Code::OK          Successfully initialized
+  //  - Code::ABORTED     Initialization was failed
+  virtual utils::Status ConfigLoadingStatus() = 0;
```

Comment:
Can we do add another function 

status LoadServiceRollouts();

It will be much easier to read.

We can add comment say, this can be called only once,
the data is from server_config.